jerry-core/CMakeLists.txt

```diff
@@ -122,20 +122,191 @@ set(INCLUDE_CORE_PRIVATE ${INCLUDE_CORE_PRIVATE} PARENT_SCOPE) # for tests/unit-
+    ecma/builtin-objects/typedarray/ecma-builtin-int8array.c
+    ecma/builtin-objects/typedarray/ecma-builtin-typedarray-helpers.c
+    ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c
+    ecma/builtin-objects/typedarray/ecma-builtin-typedarray.c
+    ecma/builtin-objects/typedarray/ecma-builtin-uint16array-prototype.c
+    ecma/builtin-objects/typedarray/ecma-builtin-uint16array.c
+    ecma/builtin-objects/typedarray/ecma-builtin-uint32array-prototype.c
+    ecma/builtin-objects/typedarray/ecma-builtin-uint32array.c
+    ecma/builtin-objects/typedarray/ecma-builtin-uint8array-prototype.c
+    ecma/builtin-objects/typedarray/ecma-builtin-uint8array.c
+    ecma/builtin-objects/typedarray/ecma-builtin-uint8clampedarray-prototype.c
+    ecma/builtin-objects/typedarray/ecma-builtin-uint8clampedarray.c
+    ecma/operations/ecma-arguments-object.c
+    ecma/operations/ecma-array-object.c
+    ecma/operations/ecma-arraybuffer-object.c
+    ecma/operations/ecma-async-generator-object.c
+    ecma/operations/ecma-big-uint.c
+    ecma/operations/ecma-bigint-object.c
+    ecma/operations/ecma-bigint.c
+    ecma/operations/ecma-boolean-object.c
+    ecma/operations/ecma-comparison.c
+    ecma/operations/ecma-container-object.c
+    ecma/operations/ecma-conversion.c
+    ecma/operations/ecma-dataview-object.c
+    ecma/operations/ecma-eval.c
+    ecma/operations/ecma-exceptions.c
+    ecma/operations/ecma-function-object.c
+    ecma/operations/ecma-get-put-value.c
+    ecma/operations/ecma-iterator-object.c
+    ecma/operations/ecma-jobqueue.c
+    ecma/operations/ecma-lex-env.c
+    ecma/operations/ecma-number-arithmetic.c
+    ecma/operations/ecma-number-object.c
+    ecma/operations/ecma-objects-general.c
+    ecma/operations/ecma-objects.c
+    ecma/operations/ecma-promise-object.c
+    ecma/operations/ecma-proxy-object.c
+    ecma/operations/ecma-reference.c
+    ecma/operations/ecma-regexp-object.c
+    ecma/operations/ecma-string-object.c
+    ecma/operations/ecma-symbol-object.c
+    ecma/operations/ecma-typedarray-object.c
+    jcontext/jcontext.c
+    jmem/jmem-allocator.c
+    jmem/jmem-heap.c
+    jmem/jmem-poolman.c
+    jrt/jrt-fatals.c
+    lit/lit-char-helpers.c
+    lit/lit-magic-strings.c
+    lit/lit-strings.c
+    parser/js/byte-code.c
+    parser/js/common.c
+    parser/js/js-lexer.c
+    parser/js/js-parser-expr.c
+    parser/js/js-parser-mem.c
+    parser/js/js-parser-module.c
+    parser/js/js-parser-statm.c
+    parser/js/js-parser-tagged-template-literal.c
+    parser/js/js-parser-util.c
+    parser/js/js-parser.c
+    parser/js/js-scanner-ops.c
+    parser/js/js-scanner-util.c
+    parser/js/js-scanner.c
+    parser/regexp/re-bytecode.c
+    parser/regexp/re-compiler.c
+    parser/regexp/re-parser.c
+    vm/opcodes-ecma-arithmetics.c
+    vm/opcodes-ecma-bitwise.c
+    vm/opcodes-ecma-relational-equality.c
+    vm/opcodes.c
+    vm/vm-stack.c
+    vm/vm-utils.c
+    vm/vm.c
+    )
```

Comment:
(style) This looks weird. But more importantly, it is used inconsistently. Sometimes it is indented by 4 spaces (like here). Sometimes it is indented by 3 spaces (so that the closing parentheses aligns with the opening one). And sometimes it is on the same line as the last element in the list.

Personally, I don't like any of them.
```cmake
set(LIST_VAR
  elem1
  elem2
  elem3
)
```
This style above is more appealing to me. (Indentation is 2 spaces, like anywhere else in the cmake files. Closing parenthesis is on a separate line to allow for clean list edits. Etc.)

However, I do realize that the "closing parenthesis on the line of the last item" is the most commonly used style throughout the files.

Reply:
I'm ok with the suggested style.